agent/tests/test_cli.py

```
@@ -1057,7 +1037,7 @@ def test_decline_init_exits(self, mock_input, mock_exec, mock_chat):
     def test_ctrl_c_during_init_prompt(self, mock_input, mock_exec, mock_chat):
         """Verify ctrl c during init prompt."""
         mock_exec.return_value = self._no_config_status()
-        result = runner.invoke(app, [])
+        result = runner.invoke(app, ["chat"])
```

Ruff (0.15.4)

[error] Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)